pkg/resourcemanager/mysql/database/client.go

```diff
@@ -11,14 +11,17 @@ import (
 	"github.com/Azure/azure-service-operator/pkg/resourcemanager/iam"
 )
 
+//MySQLDatabaseClient struct
 type MySQLDatabaseClient struct {
 }
 
+//NewMySQLDatabaseClient create a new MySQLDatabaseClient
 func NewMySQLDatabaseClient() *MySQLDatabaseClient {
 	return &MySQLDatabaseClient{}
 }
 
-func getMySQLDatabasesClient() mysql.DatabasesClient {
+//GetMySQLDatabasesClient return the mysqldatabaseclient
+func GetMySQLDatabasesClient() mysql.DatabasesClient {
```

Comment:
Minor: should this be singular `...DatabaseClient` rather than plural `...DatabasesClient`? In other places it's singular `MySQLDatabaseClient`.

Reply:
@davefellows , dave, the GetMySQLDatabasesClient() uses plural because it returns the mysql.DatabasesClient(a plural), which is provided by Azure SDK for Go. So here it follows the SDK's naming. :)